spec/components/summary_card/view_preview.rb

```diff
@@ -1,9 +1,18 @@
+require "govuk/components"
 class SummaryCard::ViewPreview < ViewComponent::Preview
-  def default_state
-    render_component(SummaryCard::View.new(title: "Personal Details", rows: {
-      "First names": "Mike",
-      "Middle names": "Larson",
-      "Last name": "Doyle",
-    }))
+  def with_no_action_links
+    render_component(SummaryCard::View.new(title: "Personal Details", rows: [
+      { key: "First names", value: "Mike" },
+      { key: "Middle names", value: "Larson" },
```

Comment:
Do we do middle names now?

Reply:
It's just an example of what we can do with the components!